source/manual/intro-to-docker.html.md

```diff
@@ -8,11 +8,14 @@ review_in: 6 months
 owner_slack: "#govuk-developers"
 ---
 
-This is a tutorial where we run a few things to get [content-publisher] up and running in Docker. If you are new to Docker, you can use this as a quick intro, in the context of the GOV.UK stack.
+We use [govuk-docker] to help us develop stuff on GOV.UK. If you're new to Docker, this will useful insights into how we use it in the context of the GOV.UK stack.
```

Comment:
```suggestion
We use [govuk-docker] to help us develop stuff on GOV.UK. If you're new to Docker, this will provide useful insights into how we use it in the context of the GOV.UK stack.
```

Reply:
Looks like you didn't push this change?

Reply:
Sorry, just saw it. Good spot - fixed 👍